src/pudl_rmi/deprish.py

```diff
@@ -366,13 +368,17 @@ def fill_in_df(self, df_to_fill, common_allocated):
             filled_df[f"book_reserve{suffix}"] = filled_df[
                 f"book_reserve{suffix}"
             ].fillna(
-                filled_df[f"plant_balance{suffix}"]
+                (filled_df[f"plant_balance{suffix}"] * (1 - filled_df.net_salvage_rate))
                 - (
                     filled_df[f"depreciation_annual_epxns{suffix}"]
                     * filled_df.remaining_life_avg
                 )
             )
-
+            filled_df[f"net_plant_balance{suffix}"] = filled_df[
```

Comment:
Kind of a nit but it seems like this function could maybe be better formatted (and maybe easier to debug later) with fewer `fillna` calls where you pass in a dictionary of columns to values for the `value` parameter and do `filled_df.fillna(value=col_to_val_dict)`. It seems like it would still need to be a couple `fillna` calls because there's an order in which the columns are filled but it might make the whole thing more readable / understandable in how these column values are calculated. Might also be faster.

Reply:
oh this is an excellent idea!

Reply:
ty @katie-lamb  I just updated this method with this option and it is sooo much nicer to look at.